src/Adapter/PlatformServices.NetCore/PlatformServices.NetCore.csproj

```diff
@@ -19,30 +19,58 @@
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\ns13DeploymentItem.cs" Link="Deployment\ns13DeploymentItem.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\ns13TestRunDirectories.cs" Link="Deployment\ns13TestRunDirectories.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Services\ns13MSTestAdapterSettings.cs" Link="Services\ns13MSTestAdapterSettings.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Services\ns13SettingsProvider.cs" Link="Services\ns13SettingsProvider.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Services\ns13TestContextPropertyStrings.cs" Link="Services\ns13TestContextPropertyStrings.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Services\ns13TestDeployment.cs" Link="Services\ns13TestDeployment.cs" />
     <Compile Include="..\PlatformServices.Shared\netstandard1.3\Services\ns13TraceListener.cs" Link="Services\ns13TraceListener.cs" />
     <Compile Include="..\PlatformServices.Shared\netstandard1.3\Services\ns13TraceListenerManager.cs" Link="Services\ns13TraceListenerManager.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Utilities\ns13DeploymentItemUtility.cs" Link="Utilities\ns13DeploymentItemUtility.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Utilities\ns13DeploymentUtilityBase.cs" Link="Utilities\ns13DeploymentUtilityBase.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Utilities\ns13FileUtility.cs" Link="Utilities\ns13FileUtility.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Utilities\ns13Validate.cs" Link="Utilities\ns13Validate.cs" />
+    <Compile Include="Deployment\AssemblyLoadWorker.cs" />
     <Compile Include="Friends.cs" />
+    <Compile Include="Utilities\AssemblyUtility.cs" />
+    <Compile Include="Utilities\DeploymentUtility.cs" />
+    <Compile Include="Utilities\RecursiveDirectoryPath.cs" />
+    <Compile Include="Utilities\ReflectionUtility.cs" />
+    <Compile Include="..\PlatformServices.Shared\netstandard1.3\Resources\Resource.Designer.cs">
+      <Link>Resources\Resource.Designer.cs</Link>
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resource.resx</DependentUpon>
+    </Compile>
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="11.0.0" />
+    <EmbeddedResource Include="..\PlatformServices.Shared\netstandard1.3\Resources\Resource.resx">
+      <Link>Resources\Resource.resx</Link>
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resource.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="15.0.0" />
```

Comment:
Please check the implications of this.

Reply:
It'll work fine as we are currently using version 15 only at runtime.